clang/lib/CConv/ConstraintVariables.cpp

```diff
@@ -1483,3 +1485,31 @@ void constrainConsVarGeq(std::set<ConstraintVariable *> &LHS,
   }
 }
 
+void PointerVariableConstraint::replaceCvars(ConstraintVariable *FromCV) {
+  PVConstraint *From = dyn_cast<PVConstraint>(FromCV);
+  assert (From != nullptr);
+  CAtoms CFrom = From->getCvars();
+  assert (vars.size() == CFrom.size());
+  vars = CFrom; // FIXME: structural copy? By reference?
+}
```

Comment:
We should also see `getFV()` if not `nullptr`, we should call `getFV()->replaceCvars(FromCV->getFV())`

Reply:
Good catch!